apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -224,7 +224,13 @@ export default function LearningGoals({
           .catch(error => console.error(error));
       });
     }
-  }, [studentLevelInfo, learningGoals, currentLearningGoal, open, productTour]);
+  }, [
+    studentLevelInfo?.user_id,
```

Comment:
This is to prevent the network call within this `useEffect` from firing multiple times when the `studentLevelInfo` object changes, as React does strict equality comparison when checking dependency changes. This ensures that this useEffect only fires if the actual user ID has changed.